session/txninfo/txn_info.go

```diff
@@ -41,6 +41,15 @@ const (
 	TxnRollingBack
 )
 
+// StateLabel is used to translate TxnRunningState to its prometheus label name.
+var StateLabel map[TxnRunningState]string = map[TxnRunningState]string{
```

Comment:
How about wrap the mapping operation in a function and make the map itself private, so that we can avoid the map being mis-changed?